lib/apr/notifications.ex

```diff
@@ -32,7 +32,7 @@ defmodule Apr.Notifications do
           Apr.Views.ConversationSlackView.render(subscription, event)
 
         "invoices" ->
-          Apr.Views.InvoiceSlackView.render(subscription, event, routing_key)
+          Apr.Views.SellerSlackView.render(subscription, event, routing_key)
```

Comment:
To support the new ["sellers"](https://github.com/artsy/gravity/blob/cab05987c4eb513788297799718320eb3e5c087a/app/presenters/event_stream/merchant_account_event.rb#L3) topic, _I think_ we'll need to

- [add a new child](https://github.com/artsy/aprd/blob/1f1f517eb6e28050bd371a968f268dff55a1d6b7/lib/apr/application.ex#L17-L68) to listen for the new topic (which will automatically create a topic exchange in RabbitMQ based on our test the other day).
- expand here and add a new "sellers" switch case and render the slack view.

It might be no harm to also migrate "invoices" topic to use the same view, and I assume we'll want to deprecate/retire the invoices topic eventually.

Reply:
Thank you for pointing it out. I forgot to make a change regarding them. I updated `notifications.ex` here: https://github.com/artsy/aprd/pull/364/commits/62fa9344477053a048b62afcc187014ec744d229